npc/instances/EndlessTower.txt

```diff
@@ -216,6 +216,9 @@ e_tower,81,105,0	script	Tower Protection Stone	2_MONEMUS,{
 	.@etower_timer = questprogress(60200,PLAYTIME); // 1 week
 	.@etower_timer2 = questprogress(60201,PLAYTIME); // 4 hours
 
+	if (has_instance2("1@tower") >= 0)
```

Comment:
It was a commit non-related to this pull, I have delete it. Thanks for the help explaining how to do it!